pages/dev/hello-world.md

```diff
@@ -103,8 +103,8 @@ In the above, we sent a message - "Hello World" - from a smart contract on Moonb
 
 The full transaction flow was:
 1. Started the local developer environment to run local EVM blockchains.
-2. The first node script deployed [this](https://github.com/axelarnetwork/axelar-local-gmp-examples/blob/main/examples/call-contract/ExecutableSample.sol) smart contract to all the EVM chains in our local developer environment.
-3. The second node script ran [this](https://github.com/axelarnetwork/axelar-local-gmp-examples/blob/main/examples/call-contract/index.js#L22) test script on local Moonbeam, which:
+2. The first node script deployed [this](https://github.com/axelarnetwork/axelar-examples/blob/main/examples/call-contract/ExecutableSample.sol) smart contract to all the EVM chains in our local developer environment.
```

Comment:
`https://github.com/axelarnetwork/axelar-examples/blob/main/examples/call-contract/ExecutableSample.sol` to `https://github.com/axelarnetwork/axelar-examples/blob/main/examples/evm/call-contract/ExecutableSample.sol`